packages/payment-integration-api/src/payment/index.ts

```diff
@@ -44,7 +45,7 @@ export { PaymentInitializeOptions } from './payment-initialize-options';
 export { PaymentRequestOptions } from './payment-request-options';
 export { default as PaymentMethod } from './payment-method';
 export { default as PaymentResponse } from './payment-response';
-export { default as PaymentResponseBody } from './payment-response-body';
+export { default as PaymentResponseBody, AdditionalActionRequired } from './payment-response-body';
```

Comment:
Is `AdditionalActionRequired` used in this PR?

Reply:
Sharp eye, thanks! 🫶🏻
Fixed✅